gpu-allocation.md

```diff
@@ -0,0 +1,17 @@
+## GPU Allocation
+
+Kubecost performs GPU allocation through [container resource limits](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/), which are parsed for each pod which has a container resource request or limit of type `nvidia.com/gpu`. A pod requesting GPU resources will have a field `spec.containers.resources` as follows:
```

Comment:
Should "container resource limits" be something like the "container resource block"? Based on the description, looks like either requests or limits can be used. 

Also, silly question.. can you today have a pod that uses GPU without a GPU request or limit?

Reply:
Yeah, it's true that either requests or limits are used. I think what I was thinking there was that you can have a limit without a request, but not vice-versa, but you're right that being clearer is better.

As for a pod using a GPU without a request... no? I think? You can have a pod with an image that needs a GPU to run e.g. a CUDA program, but without that request/limit it's not going to be using the [device plugin ds](https://github.com/NVIDIA/k8s-device-plugin) and will just error + crashloop as if a gpu doesn't exist.